### PR TITLE
Update Tekkon Technology URL & Add Ishani Technology Pvt. Ltd.

### PR DESCRIPTION
**Pull request by azens1995**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 List of tech companies (or teams) that are based or working in Nepal. :nepal:
 
-|     |     |     |     |     |     |     |     |     |
-|:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |
+|                 |                 |                 |                 |                 |                 |                 |                 |                 |
```
Comment (mesaugat) [inline README.md:5]: @azens1995 Thank you for the PR. Can you revert this change? I think you are using a code formatted like prettier?

File: README.md [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
 
 ### Website: https://airtable.com/shrXv3YvlfxLnHJfr
 
 ### Adding a new Company to the List
 
-* [Fork](https://guides.github.com/activities/forking) the repository
-* Add your company in alphabetical order using the mentioned format below
-* Company description should be short and should not end with a period
-* Here's a good [example](https://github.com/mesaugat/tech-companies-in-nepal/pull/7/files)
+- [Fork](https://guides.github.com/activities/forking) the repository
```
Comment (mesaugat) [inline README.md:13]: Same here.

File: README.md (hunk 1 above)
Comment (azens1995) [reply README.md:5]: Yes, @mesaugat dai. These are changes by the prettier. OK. I will revert those changes back.

File: README.md [hunk 3]
```diff
@@ -144,7 +145,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 
 - [PageVamp](https://www.pagevamp.com/) | Sanepa, Lalitpur | Create and Update Website from Facebook Page
 - [Parewa Labs Pvt. Ltd.](https://parewalabs.com/) | Lazimpat, Kathmandu | Make products for
-Programming Enthusiasts
+  Programming Enthusiasts
```
Comment (mesaugat) [inline README.md:148]: Move this to the above line.

```
Make products for Programming Enthusiasts
```

File: README.md [hunk 4]
```diff
@@ -196,15 +198,15 @@ Programming Enthusiasts
...
+- [Visualise VR](http://visualisevr.com/) | Pulchowk, Lalitpur | Help business benefit from mixed reality technologies
 - [Viveka Health](http://www.vivekahealth.com/) | Sanepa, Lalitpur | Software company working on health data projects
 - [Walkers Hive Pvt. Ltd.](https://walkershive.com/) | Putalisadak, Kathmandu | Digital Marketing, Optimized Website, Management System, Mobile Applications, Software Integration, Advanced Level Programming Course, IT Consultation
 - [Webpage Nepal Pvt. Ltd.](http://webpagenepal.com/) | New Road, Pokhara | Pokhara based web development company
 - [WebRobo](https://www.web-robo.com/) | Setopul, Kathmandu | Creative website development and digital marketing
 - [WebSoft Nepal Pvt. Ltd.](http://www.websoftnepal.com) | Indradev Marg, Narayangarh | Provides a complete solution on Web Design and Hostings
 - [Worldlink Communications Pvt. Ltd.](https://worldlink.com.np/) | Jawalakhel, Lalitpur | Complete IT service provider providing services such as data connectivity, network integration and consulting, web hosting, hardware sales and maintenance
 
-## Y - \#
```
Comment (mesaugat) [inline README.md:207]: Revert this change too.
